<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1590</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>1301</width>
     <height>541</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <column>
    <property name="text">
     <string>타임스탬프</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>주문 상태</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>계약자</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>연락처</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고(고객메모)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고(직원메모)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>주문번호(ESM)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>제품사</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>모델명</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>last_access</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="find_order_state">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>조회</string>
   </property>
  </widget>
  <widget class="QComboBox" name="order_state">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>등록 대기</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>예약</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>등록 완료</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>취소</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>설치 보류</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>설치 완료</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>재확인 요청</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="find_one_hundred">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>80</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>최근 100개 조회</string>
   </property>
  </widget>
  <widget class="QPushButton" name="all_find_btn">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>조회</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="all_text">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>16</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>10</y>
     <width>20</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>20</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>최대 1만개 조회</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>주문상태 조회</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>50</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>전체 조회</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>180</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>오류 박스</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>210</y>
     <width>211</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
